ansible/templates/nginx.conf.j2

```diff
@@ -59,7 +59,7 @@ server {
 server {
   server_name {{domain}};
   listen 80;
-  return 301 https://{{fqdn}}$request_uri;
+  return 301 https://{{canonical_domain}}$request_uri;
```

Comment:
Should this be `domain`? If it's `canonical_domain`, we can probably get rid of all the non-canonical domain `ssl` server blocks in production. I guess I missed this in my initial PR.